### PR TITLE
add events.retroscene.org production link type

### PR DESCRIPTION
**Pull request by akanyuk**

### CODE REVIEW

File: demoscene/utils/groklinks.py [hunk 1]
```diff
@@ -1710,6 +1710,14 @@ def __str__(self):
         return u"https://diskmag.conspiracy.hu/magazine/%s/#/edition=%s" % (magazine, edition)
 
 
+class EventsRetrosceneRelease(AbstractBaseUrl):
+    site =  Site("Events", long_name="Retroscene events", icon_path='images/icons/external_sites/events_retroscene.png')
```
Comment (gasman) [inline demoscene/utils/groklinks.py:1714]: You haven't specified a `url` or `allowed_hostnames` argument here, so this ends up matching every URL, not just ones on events.retroscene.org.

**Review by gasman — CHANGES_REQUESTED** [review]

File: demoscene/utils/groklinks.py (hunk 1 above)
Comment (akanyuk) [reply demoscene/utils/groklinks.py]: Added `url` argument

**Comment by gasman** [comment]
Afraid this still isn't quite right - for productions that have both a download link and an external link pointing to events.retroscene.org (such as https://demozoo.org/graphics/297092/), both of them match the regexp and get converted to external links. We probably need to make the regexp stricter, so that it only matches paths that do NOT start with `files`, and end in a number.

File: demoscene/utils/groklinks.py [hunk 2]
```diff
@@ -1710,6 +1710,18 @@ def __str__(self):
         return u"https://diskmag.conspiracy.hu/magazine/%s/#/edition=%s" % (magazine, edition)
 
 
+class EventsRetrosceneRelease(AbstractBaseUrl):
+    site =  Site(
+        "Events", long_name="Retroscene events",
```
Comment (gasman) [inline demoscene/utils/groklinks.py:1715]: Also, "Events" isn't really descriptive enough as link text - I'd suggest using "Retroscene events" as the standard label, not just the long name.
```suggestion
        "Retroscene events",
```

**Comment by gasman** [comment]
Fixed and merged in c95b65b8fa792989961828c48e13e1870387f9ec - thanks!

**Pull request closed without merging**